src/bootstrap.js

```diff
@@ -4,9 +4,35 @@
 
 /* eslint-disable no-unused-vars */
 
+const { utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/Services.jsm");
+
 function startup({webExtension}, reason) {
-  webExtension.startup().then(() => {
+  Services.telemetry.registerEvents("lockbox", {
+    "startup": {
+      methods: ["startup"],
+      objects: ["addon", "webextension"],
+    },
+    "click": {
+      methods: ["click"],
+      objects: ["browser_action"],
+    },
+  });
+
+  webExtension.startup().then(({browser}) => {
     console.log("embedded webextension has started");
+    Services.telemetry.recordEvent("lockbox", "startup", "webextension");
```

Comment:
(cc: @irrationalagent )

Do you want the user identifier (if available) on startup?

Reply:
Assuming the uid is desired, I think this needs to be ported into `webextension/background/index.js`.